lib/mix_docker.ex

```diff
@@ -80,15 +88,19 @@ defmodule MixDocker do
     Application.get_env(:mix_docker, :image) || to_string(Mix.Project.get.project[:app])
   end
 
-  defp image_tag(:version) do
+  defp image_tag(version: version_template) do
     version = Mix.Project.get.project[:version]
     count   = git_commit_count()
     sha     = git_head_sha()
 
-    "#{version}.#{count}-#{sha}"
+    Code.eval_string(version_template, mix_version: version, git_count: count, git_sha: sha) |> elem(0)
```

Comment:
It should be simple replace "${var}" with value, no need to use dangerous eval_string here.